[rospy.client][INFO] 2020-02-05 22:08:47,356: init_node, name[/monitor], pid[16837]
[xmlrpc][INFO] 2020-02-05 22:08:47,357: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-05 22:08:47,358: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:37825/]
[rospy.impl.masterslave][INFO] 2020-02-05 22:08:47,359: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:37825/
[rospy.init][INFO] 2020-02-05 22:08:47,363: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:37825/]
[rospy.registration][INFO] 2020-02-05 22:08:47,364: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-02-05 22:08:47,365: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-02-05 22:08:47,463: registered with master
[rospy.rosout][INFO] 2020-02-05 22:08:47,464: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-05 22:08:47,470: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-05 22:08:47,472: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-05 22:08:47,580: topic[/whycon/poses] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/], count 0
[rospy.internal][INFO] 2020-02-05 22:08:47,624: topic[/led_control_topic] adding connection to [/serial_node], count 0
[rospy.internal][INFO] 2020-02-05 22:08:47,678: topic[/stats_sr] adding connection to [/rostopic_16570_1580920716253], count 0
[rospy.internal][INFO] 2020-02-05 22:08:47,703: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-02-05 22:08:47,747: topic[/detection_info] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:35297/], count 0
[rospy.core][INFO] 2020-02-05 22:11:47,660: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-02-05 22:11:47,810: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-05 22:11:47,813: topic[/stats_sr] removing connection to /rostopic_16570_1580920716253
[rospy.internal][INFO] 2020-02-05 22:11:47,813: topic[/led_control_topic] removing connection to /serial_node
[rospy.internal][INFO] 2020-02-05 22:11:47,814: topic[/whycon/poses] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/
[rospy.internal][INFO] 2020-02-05 22:11:47,814: topic[/detection_info] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:35297/
[rospy.impl.masterslave][INFO] 2020-02-05 22:11:47,814: signal-2
[rospy.core][INFO] 2020-02-05 22:11:47,926: signal_shutdown [atexit]
[rospy.client][INFO] 2020-02-05 22:21:07,873: init_node, name[/monitor], pid[18565]
[xmlrpc][INFO] 2020-02-05 22:21:07,873: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-05 22:21:07,873: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:39105/]
[rospy.init][INFO] 2020-02-05 22:21:07,874: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:39105/]
[rospy.impl.masterslave][INFO] 2020-02-05 22:21:07,874: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:39105/
[rospy.registration][INFO] 2020-02-05 22:21:07,875: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-02-05 22:21:07,875: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-02-05 22:21:07,974: registered with master
[rospy.rosout][INFO] 2020-02-05 22:21:07,975: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-05 22:21:07,985: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-05 22:21:07,989: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-05 22:21:08,092: topic[/whycon/poses] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/], count 0
[rospy.internal][INFO] 2020-02-05 22:21:08,120: topic[/led_control_topic] adding connection to [/serial_node], count 0
[rospy.internal][INFO] 2020-02-05 22:21:08,147: topic[/stats_sr] adding connection to [/rostopic_18238_1580921372508], count 0
[rospy.internal][INFO] 2020-02-05 22:21:08,215: topic[/detection_info] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:40933/], count 0
[rospy.internal][INFO] 2020-02-05 22:21:08,252: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-02-05 22:21:43,109: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2020-02-05 22:21:43,116: Unable to initiate TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:44639 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:40933/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-02-05 22:21:43,262: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-05 22:21:43,263: topic[/stats_sr] removing connection to /rostopic_18238_1580921372508
[rospy.internal][INFO] 2020-02-05 22:21:43,264: topic[/led_control_topic] removing connection to /serial_node
[rospy.internal][INFO] 2020-02-05 22:21:43,269: topic[/whycon/poses] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/
[rospy.internal][INFO] 2020-02-05 22:21:43,269: topic[/detection_info] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:40933/
[rospy.impl.masterslave][INFO] 2020-02-05 22:21:43,270: signal-2
[rospy.core][INFO] 2020-02-05 22:21:43,370: signal_shutdown [atexit]
[rospy.client][INFO] 2020-02-05 22:21:51,504: init_node, name[/monitor], pid[18773]
[xmlrpc][INFO] 2020-02-05 22:21:51,504: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-05 22:21:51,507: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:44315/]
[rospy.init][INFO] 2020-02-05 22:21:51,507: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:44315/]
[rospy.impl.masterslave][INFO] 2020-02-05 22:21:51,507: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:44315/
[xmlrpc][INFO] 2020-02-05 22:21:51,508: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-02-05 22:21:51,508: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-02-05 22:21:51,608: registered with master
[rospy.rosout][INFO] 2020-02-05 22:21:51,608: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-05 22:21:51,616: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-05 22:21:51,618: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-05 22:21:51,724: topic[/whycon/poses] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/], count 0
[rospy.internal][INFO] 2020-02-05 22:21:51,731: topic[/stats_sr] adding connection to [/rostopic_18238_1580921372508], count 0
[rospy.internal][INFO] 2020-02-05 22:21:51,769: topic[/led_control_topic] adding connection to [/serial_node], count 0
[rospy.internal][INFO] 2020-02-05 22:21:51,842: topic[/detection_info] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:46261/], count 0
[rospy.internal][INFO] 2020-02-05 22:21:51,918: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-02-05 22:21:52,910: signal_shutdown [signal-2]
[rospy.internal][WARNING] 2020-02-05 22:21:52,918: Unknown error initiating TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:33163 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:46261/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rosgraph/network.py", line 362, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-02-05 22:21:52,919: topic[/detection_info] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:46261/
[rospy.internal][INFO] 2020-02-05 22:21:53,071: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-05 22:21:53,072: topic[/stats_sr] removing connection to /rostopic_18238_1580921372508
[rospy.internal][INFO] 2020-02-05 22:21:53,072: topic[/led_control_topic] removing connection to /serial_node
[rospy.internal][INFO] 2020-02-05 22:21:53,073: topic[/whycon/poses] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/
[rospy.internal][WARNING] 2020-02-05 22:21:53,073: Unknown error initiating TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:44807 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-02-05 22:21:53,074: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2020-02-05 22:21:53,074: signal-2
[rospy.client][INFO] 2020-02-05 22:22:54,380: init_node, name[/monitor], pid[19021]
[xmlrpc][INFO] 2020-02-05 22:22:54,381: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-05 22:22:54,381: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:41421/]
[rospy.init][INFO] 2020-02-05 22:22:54,381: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:41421/]
[rospy.impl.masterslave][INFO] 2020-02-05 22:22:54,381: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:41421/
[rospy.registration][INFO] 2020-02-05 22:22:54,382: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-02-05 22:22:54,382: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-02-05 22:22:54,482: registered with master
[rospy.rosout][INFO] 2020-02-05 22:22:54,482: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-05 22:22:54,486: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-05 22:22:54,489: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-05 22:22:54,559: topic[/whycon/poses] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/], count 0
[rospy.internal][INFO] 2020-02-05 22:22:54,571: topic[/led_control_topic] adding connection to [/serial_node], count 0
[rospy.internal][INFO] 2020-02-05 22:22:54,575: topic[/stats_sr] adding connection to [/rostopic_18238_1580921372508], count 0
[rospy.internal][INFO] 2020-02-05 22:22:54,682: topic[/detection_info] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:37309/], count 0
[rospy.internal][INFO] 2020-02-05 22:22:54,763: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-02-05 22:24:03,724: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2020-02-05 22:24:03,728: Unable to initiate TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:40131 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:37309/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-02-05 22:24:03,888: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-05 22:24:03,888: topic[/stats_sr] removing connection to /rostopic_18238_1580921372508
[rospy.internal][INFO] 2020-02-05 22:24:03,888: topic[/led_control_topic] removing connection to /serial_node
[rospy.internal][INFO] 2020-02-05 22:24:03,889: topic[/whycon/poses] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/
[rospy.internal][WARNING] 2020-02-05 22:24:03,889: Unknown error initiating TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:44807 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-02-05 22:24:03,890: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.internal][INFO] 2020-02-05 22:24:03,890: topic[/detection_info] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:37309/
[rospy.impl.masterslave][INFO] 2020-02-05 22:24:03,890: signal-2
[rospy.core][INFO] 2020-02-05 22:24:03,991: signal_shutdown [atexit]
[rospy.client][INFO] 2020-02-05 22:26:28,110: init_node, name[/monitor], pid[19402]
[xmlrpc][INFO] 2020-02-05 22:26:28,111: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-05 22:26:28,112: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:37461/]
[rospy.init][INFO] 2020-02-05 22:26:28,112: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:37461/]
[rospy.impl.masterslave][INFO] 2020-02-05 22:26:28,112: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:37461/
[rospy.registration][INFO] 2020-02-05 22:26:28,113: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-02-05 22:26:28,113: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-02-05 22:26:28,212: registered with master
[rospy.rosout][INFO] 2020-02-05 22:26:28,213: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-05 22:26:28,220: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-05 22:26:28,222: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-05 22:26:28,310: topic[/led_control_topic] adding connection to [/serial_node], count 0
[rospy.internal][INFO] 2020-02-05 22:26:28,329: topic[/whycon/poses] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/], count 0
[rospy.internal][INFO] 2020-02-05 22:26:28,344: topic[/stats_sr] adding connection to [/rostopic_19368_1580921784818], count 0
[rospy.internal][INFO] 2020-02-05 22:26:28,493: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-02-05 22:26:28,545: topic[/detection_info] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:35807/], count 0
[rospy.core][INFO] 2020-02-05 22:27:48,995: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2020-02-05 22:27:49,002: Unable to initiate TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:43299 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:35807/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-02-05 22:27:49,154: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-05 22:27:49,155: topic[/stats_sr] removing connection to /rostopic_19368_1580921784818
[rospy.internal][INFO] 2020-02-05 22:27:49,155: topic[/led_control_topic] removing connection to /serial_node
[rospy.internal][INFO] 2020-02-05 22:27:49,156: topic[/whycon/poses] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/
[rospy.internal][INFO] 2020-02-05 22:27:49,156: topic[/detection_info] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:35807/
[rospy.impl.masterslave][INFO] 2020-02-05 22:27:49,157: signal-2
[rospy.internal][WARNING] 2020-02-05 22:27:49,159: Unknown error initiating TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:44807 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.core][INFO] 2020-02-05 22:27:49,259: signal_shutdown [atexit]
[rospy.client][INFO] 2020-02-05 22:27:53,053: init_node, name[/monitor], pid[19670]
[xmlrpc][INFO] 2020-02-05 22:27:53,053: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-05 22:27:53,054: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:38429/]
[rospy.init][INFO] 2020-02-05 22:27:53,054: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:38429/]
[rospy.impl.masterslave][INFO] 2020-02-05 22:27:53,054: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:38429/
[rospy.registration][INFO] 2020-02-05 22:27:53,056: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-02-05 22:27:53,056: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-02-05 22:27:53,155: registered with master
[rospy.rosout][INFO] 2020-02-05 22:27:53,155: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-05 22:27:53,161: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-05 22:27:53,162: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-05 22:27:53,224: topic[/whycon/poses] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/], count 0
[rospy.internal][INFO] 2020-02-05 22:27:53,289: topic[/led_control_topic] adding connection to [/serial_node], count 0
[rospy.internal][INFO] 2020-02-05 22:27:53,292: topic[/detection_info] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:43969/], count 0
[rospy.internal][INFO] 2020-02-05 22:27:53,312: topic[/stats_sr] adding connection to [/rostopic_19368_1580921784818], count 0
[rospy.internal][INFO] 2020-02-05 22:27:53,385: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-02-05 22:27:53,487: topic[/stats_sr] adding connection to [/rosbag_record_beacon_detections], count 1
[rospy.internal][INFO] 2020-02-05 22:28:18,027: topic[/stats_sr] removing connection to /rosbag_record_beacon_detections
[rospy.core][INFO] 2020-02-05 22:28:28,152: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2020-02-05 22:28:28,159: Unable to initiate TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:33395 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:43969/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-02-05 22:28:28,319: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-05 22:28:28,320: topic[/stats_sr] removing connection to /rostopic_19368_1580921784818
[rospy.internal][INFO] 2020-02-05 22:28:28,320: topic[/led_control_topic] removing connection to /serial_node
[rospy.internal][INFO] 2020-02-05 22:28:28,321: topic[/whycon/poses] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/
[rospy.internal][INFO] 2020-02-05 22:28:28,321: topic[/detection_info] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:43969/
[rospy.impl.masterslave][INFO] 2020-02-05 22:28:28,321: signal-2
[rospy.core][INFO] 2020-02-05 22:28:28,936: signal_shutdown [atexit]
[rospy.client][INFO] 2020-02-05 22:28:46,756: init_node, name[/monitor], pid[19919]
[xmlrpc][INFO] 2020-02-05 22:28:46,756: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-02-05 22:28:46,757: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:46117/]
[rospy.init][INFO] 2020-02-05 22:28:46,757: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:46117/]
[rospy.impl.masterslave][INFO] 2020-02-05 22:28:46,757: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:46117/
[rospy.registration][INFO] 2020-02-05 22:28:46,758: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-02-05 22:28:46,758: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-02-05 22:28:46,857: registered with master
[rospy.rosout][INFO] 2020-02-05 22:28:46,857: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-05 22:28:46,864: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-05 22:28:46,865: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-05 22:28:46,929: topic[/whycon/poses] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/], count 0
[rospy.internal][INFO] 2020-02-05 22:28:47,036: topic[/led_control_topic] adding connection to [/serial_node], count 0
[rospy.internal][INFO] 2020-02-05 22:28:47,051: topic[/stats_sr] adding connection to [/rostopic_19883_1580921921768], count 0
[rospy.internal][INFO] 2020-02-05 22:28:47,118: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-02-05 22:28:47,146: topic[/detection_info] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:44901/], count 0
[rospy.internal][INFO] 2020-02-05 22:28:47,219: topic[/stats_sr] adding connection to [/rosbag_record_beacon_detections], count 1
[rospy.internal][INFO] 2020-02-05 22:29:11,731: topic[/stats_sr] removing connection to /rosbag_record_beacon_detections
[rospy.core][INFO] 2020-02-05 22:29:20,004: signal_shutdown [signal-2]
[rospy.internal][ERROR] 2020-02-05 22:29:20,008: Unable to initiate TCP/IP socket to itachi-HP-Pavilion-Laptop-15-cc1xx:38453 (http://itachi-HP-Pavilion-Laptop-15-cc1xx:44901/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-02-05 22:29:20,157: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-02-05 22:29:20,157: topic[/stats_sr] removing connection to /rostopic_19883_1580921921768
[rospy.internal][INFO] 2020-02-05 22:29:20,158: topic[/led_control_topic] removing connection to /serial_node
[rospy.internal][INFO] 2020-02-05 22:29:20,158: topic[/whycon/poses] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:34827/
[rospy.internal][INFO] 2020-02-05 22:29:20,158: topic[/detection_info] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:44901/
[rospy.impl.masterslave][INFO] 2020-02-05 22:29:20,159: signal-2
[rospy.core][INFO] 2020-02-05 22:29:20,275: signal_shutdown [atexit]
